examples/medical-center/Ballerina.toml

```diff
@@ -2,7 +2,14 @@
 org = "foo"
 name = "medical_center"
 version = "0.1.0"
-distribution = "2201.3.0"
+distribution = "2201.4.0"
 
 [build-options]
 observabilityIncluded = false
+
+[persist.entity.storage.mysql]
+host = "localhost"
+port = 3308
+user = "root"
+password = "Test123#"
+database = "medical_center_db"
```

Comment:
```suggestion
database = "medical_center_db"

```

Reply:
updated in [29c72ce](https://github.com/ballerina-platform/persist-tools/pull/120/commits/29c72ce16d2edf4882433055270167e57360d0f7)